test/Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.Test/EntityFrameworkCoreDataProtectionBuilderExtensionsTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.DataProtection.EntityFrameworkCore;
+using Microsoft.AspNetCore.DataProtection.KeyManagement;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.AspNetCore.DataProtection.EntityFrameworkCore.Test
+{
+    public class EntityFrameworkCoreDataProtectionBuilderExtensionsTests
+    {
+        [Fact]
+        public void PersistKeysToEntityFrameworkCore_UsesEntityFrameworkCoreXmlRepository()
+        {
+            var serviceCollection = new ServiceCollection();
+            serviceCollection
+                .AddDbContext<DataProtectionKeyContext>()
+                .AddDataProtection()
+                .PersistKeysToDbContext<DataProtectionKeyContext>();
+            var serviceProvider = serviceCollection.BuildServiceProvider();
```

Comment:
Can you change this to `.BuildServiceProvider(validateScopes: true)`? This typically helps us find issues in the CI services.